webui/ocw/lib/vault.py

```diff
@@ -90,6 +101,35 @@ def renew(self):
         self.revoke()
         self.getData()
 
+    def getAuthCacheFile(self):
+        oldmask = os.umask(0o077)
+        path = Path('/tmp/pcw/{}/{}'.format(self.__class__.__name__, self.namespace))
+        path.mkdir(mode=0o700, parents=True, exist_ok=True)
+        f = path.joinpath('auth.json')
+        f.touch(mode=0o600, exist_ok=True)
+        oldmask = os.umask(oldmask)
+        return f
+
+    def loadFileCache(self):
+        try:
+            filename = self.getAuthCacheFile()
+            with filename.open() as f:
```

Comment:
```suggestion
            with authjson.open() as authjson_file:
```

Reply:
I will keep `f` it's a 2 line with statement and `f` is quite common here